src/pjs/pjs.proto

```diff
@@ -0,0 +1,184 @@
+syntax = "proto3";
+
+package pjs;
+option go_package = "github.com/pachyderm/pachyderm/v2/src/pjs";
+
+import "google/protobuf/any.proto";
+
+message Job {
+  int64 id = 1;
+}
```

Comment:
I feel like embedded messages with one field have caused a lot of pain over the years.  They provide 2 different invalid states; nil Job, and non-nil Job but id = 0.  We should only have one state for unknown/unset job ids.  In that case, I feel like having `int64 job_id` in each message that references a job would be adequate; 0 indicates "the caller forgot to set it".

If we want to keep the embedded message, we should decide right now what the "unset" state is.  Either this should have a validation rule that id is not equal to 0, or each field that takes a job_id as input should have a Job != nil validation rule (`[(validate.rules).message.required = true];`).